### ISSUE TITLE
Part 2 Issues

### ISSUE THREAD

**Issue by laderast** [laderast/ready_for_r_labs#5]
Let me know about any bugs/issues with Part 2 here.

Please include the following in your comment.

a. Location in notebook (give line number): 

b. Description of Bug or Error: 

c. Suggested Fix: This could be code, spelling correction, or an edit. Be sure to include line numbers.

**Comment by ianrmcdonald** [comment]
In part2.Rmd, 

Line 68 has an incomplete sentence "It is located where the `project.rproj` file."

In the section that begins on...

Line 176:  I would say more about the distinction between observations and variables.  These students may be familiar with the idea of "unit of analysis" and in my experience the table's observation should conform to that.  Also: the distinction can get muddy because of time variables; instances of time sometimes appear as variables and the data are wide when they should be long.  But not always. The basic point:  the distinction isn't always obvious.

A general point about Excel/Sheets:  much of the work in Excel involves formatting and presentation, which is the reason they can be too messy for R.  And that discussion is a good opportunity to emphasize R's reproducibility and transparency ("here's how I generated this result"), which is a particular weakness for spreadsheets.

Line 186:  If I didn't know much about Transpose in Excel, I would think it solves every kind of formatting issue from the way it's described here.  It does solve a particular problem in one situation but I would specify it more...seems like it's being oversold.

Line 415:  In the GGPLOT2 section: it's really hard to explain and understand the whole aes() business.  I think Kiernan Healy's Data Visualization does a good job with that, and you might direct students there.  I think there's still a free version online.

Line 462:  My RStudio session started crashing at this point and I don't know why.  I didn't catch where you discussed the color parameter; good chance to remind folks that "blue" does equal the word blue without quotes.

Line 602:  R bootcamp is a great suggestion.

**Comment by laderast** [comment]
I appreciate the suggestions. Just FYI, I *can't* change the content too much (mostly edits for clarity) because I don't have time to rerecord the videos.

**Comment by ianrmcdonald** [comment]
Thanks Ted!  So have you taught that class for the last time?

**Comment by laderast** [comment]
Yes, that's the last time I'll ever teach it, at least in this form. The course will continue, just not taught by me.
